integration_tests/tests/consistency/consistency_user_unsubscribes.sql

```diff
@@ -0,0 +1,52 @@
+
+{{ config(
+    tags="fivetran_validations",
+    enabled=var('fivetran_validation_tests_enabled', false)
+) }}
+
+
+with prod as (
+
+    select 
+        1 as join_key, 
```

Comment:
That's a good question, and actually was my original plan, but obviously because of the test failure I chose to use 1 as the `join_key`. 

I guess the big question is do we believe future changes here will always result in a similar number of records before and after a change.  Since `unique_user_key` is the primary key of the old history schema with multiple records, it's quite likely any future changes here could result in deltas of channels/message types. But I'm open to switching it to this format if we believe it to be more robust. 

I do think for the new schema, this logic is worth creating the test you propose, using `_fivetran_user_id` as the grouped field. What do you think?

Reply:
After discussing in standup, moving `unique_user_key` to be the grain of this test!